src/analyses/call_graph.cpp

```diff
@@ -121,6 +121,27 @@ void call_grapht::output_dot(std::ostream &out) const
   out << "}\n";
 }
 
+
+void call_grapht::output_dot(
+  goto_functionst const &functions,
```

Comment:
Yes, this shows up an <outdated> block for me-- or see the Files tab for confirmation